docs/web/sdk-services-overview.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Key Concepts
```

Comment:
I think i prefer the name `Services Overview` instead? What do you think? Helps to differentiate between components, services, events, which all have a `Key Concepts` page.

Reply:
Mhmm... I thought about Services Concepts, Events Concepts, and Component Concepts, but the content I've put it in the pages doesn't fit. Services Overview, Events Overview, and Component Overview better capture that the content is meant to give you a description and launching pad for that element of the SDK

Reply:
Renamed the events, components, and services pages to the label "Overview" in the sidebar with the title "Components Overview", etc... for the article itself